CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Set default locale for google address form as GBR
```

Comment:
Please add the pattern following the Keep Changelog rules:

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)